web/serve.py

```diff
@@ -1,15 +1,16 @@
-# This simple app performs simple calculations and stores the results in an
```

Comment:
I renamed the file from `app.py` to `serve.py` this is mainly for pedagogical reasons, as I've found students get confused if the directory and main python file are the same name.